ipaullly/sendIT
# [#161954076] Add created delivery order in the response body

#### What does this PR do?
Adds the payload or parcel info into the response body so that the user can see the created order and the response message 
#### Description of Task to be completed?
- Ensure that the `create_order()` method in the `Parcels` class returns a payload after adding new order info into the database.
- include the payload in the response message of the `POST` new parcel view function.
#### How should this be manually tested?
clone repo into a local directory
`git clone --single-branch -b bg-POST-parcel-161954076 git@github.com:ipaullly/sendIT.git`
cd into the sendIT directory
activate virtual environment
`source venv/bin/activate`
install dependencies
`pip install requirements.txt`
power up Postman and send JSON with order details via a POST method to your server and you will see the created order within the body of the response message
#### What are the relevant pivotal tracker stories?
#22
